pravega/pravega
# Issue:529 Exposed more values in application conf to system environement variables

**Change log description**
Exposed these configurations to be set by env variables:

- controller.server.port
- controller.server.cluster
- controller.server.hostMonitorEnabled
- controller.server.store.host.containerCount

**Purpose of the change**
Issue https://github.com/pravega/pravega/issues/529

**What the code does**
Allows the 4 values to be set by env varaibles and default values in reference.conf file.

**How to verify it**
Start container by setting these env variables and view logs of controller docker container to see set values.
